.github/workflows/publish-production.yaml

```diff
@@ -36,7 +36,13 @@ jobs:
         fi
 
         curl -O -L https://raw.githubusercontent.com/replicatedhq/kots/${KOTS_TAG}/.image.env
+
         export $(cat .image.env | sed 's/#.*//g' | xargs)
+        # Postgres tag variable was changed after 1.77.0 release
+        if [ -z "$POSTGRES_ALPINE_TAG" ] && [ -n "$POSTGRES_14_TAG" ]; then
```

Comment:
do we even need this check? why not just use POSTGRES_14_TAG? POSTGRES_ALPINE_TAG doesn’t exist anymore

Reply:
we could build an older release, also the release with this var name does not exist yet.

Reply:
also, that is what it does :)